Header header
octomap_msgs/Octomap[] octomaps
string owner
uint8 num_octomaps
